src/cephadm/cephadm

```diff
@@ -3321,7 +3322,10 @@ def registry_login(url, username, password):
         out, _, _ = call_throws([container_path, 'login',
                                    '-u', username,
                                    '-p', password,
+                                   '--authfile=/etc/ceph/podman-auth.json' if 'podman' in container_path else '',
```

Comment:
I just realized that we havent' got "etc/ceph" folder in cluster hosts ( except bootstrap node). This location is available in all the containers, but mapped to /var/lib/ceph.
So I suggest to change it to var/lib/ceph

Reply:
isn't the inclusion of the authfile dependent on two things - podman with registry auth set?